orchagent/portsorch.cpp

```diff
@@ -1029,6 +1029,30 @@ void PortsOrch::updateDbPortOperStatus(sai_object_id_t id, sai_port_oper_status_
     }
 }
 
+sai_port_oper_status_t PortsOrch::getDbPortOperStatus(sai_object_id_t id)
```

Comment:
where does this one get used?

Reply:
This to get the last know port oper status saved in appDB.  Baseline for status change check.

Reply:
I have the same question. It is not called by anywhere.

---
In reply to: [209355244](https://github.com/Azure/sonic-swss/pull/557#discussion_r209355244) [](ancestors = 209355244)

Reply:
it was in old implementation, which had been superseded by m_oper_status in port class.

Reply:
Sorry I think there is miscommunication. I mean in this PR, the new function is added and seems good, but not be called by any place, so it takes no effect.